src/reuse/modules/common/userInteraction.ts

```diff
@@ -15,28 +15,29 @@ export class UserInteraction {
    * @function fillActive
    * @memberOf common.userInteraction
    * @description Fills the active input with the given value.
-   * @param {String} value - The value with witch the input should be filled.
+   * @param {String | Number | Boolean} value - The value to enter.
    * @example await common.userInteraction.fillActive("My Value");
    */
-  async fillActive(value: string) {
+  async fillActive(value: string | number | boolean) {
     const vlf = this.vlf.initLog(this.fillActive);
-    if (!(typeof value === "number" || typeof value === "string")) {
-      throw new Error("Function 'fillActive' failed: Please provide a value(datatype - number/string) as argument.");
-    } else {
+
+    if (typeof value === "number" || typeof value === "string" || typeof value === "string") {
```

Comment:
duplicate string here - but as previous comment - it probably can only be number/string?

Reply:
thanks will remove the duplicate check